packages/mui-joy/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -932,7 +932,7 @@ describe('Joy <Autocomplete />', () => {
     it('should ignore keydown event until the IME is confirmed', function test() {
       // TODO: Often times out in Firefox 78.
       // Is this slow because of testing-library or because of the implementation?
-      this.timeout(4000);
+      this?.timeout?.(4000);
```

Comment:
Is vitest fast enough that we don't need an increased timeout?

Reply:
Haven't seen it timeout yet. Will add it back for vitest if it's a problem. Otherwise we'll remove this altogether

Reply:
I've removed similar code from X without any issue too, so I guess it might have been a mocha issue? 🤔